src/Stores/Politicians/actions.js

```diff
@@ -0,0 +1,31 @@
+/* @flow */
+import type { Dispatch, PoliticianType } from '../../types';
+import { getPoliticians as apiGetPoliticians } from '../../utils/api';
+
+const getPoliticians = () => ({ type: 'POLITICIANS_LOAD' });
```

Comment:
Am I right in assuming that this is used to display a loading message when calling the "get politicians" endpoint?
If yes could it be given a bit more descriptive name, e.g. getPoliticiansLoading[Screen]? It seems to not fit with the more descriptive getPoliticiansSuccess/Failure function names.